google/cloud/spanner/samples/samples.cc

```diff
@@ -1251,6 +1251,41 @@ void QueryNewColumn(google::cloud::spanner::Client client) {
 }
 //! [END spanner_query_data_with_new_column]
 
+void ProfileQuery(google::cloud::spanner::Client client) {
+  namespace spanner = ::google::cloud::spanner;
+
+  //! [profile-query]
```

Comment:
A general question that I have also been struggling with:  Is it better for the snippet to include any necessary `using` declarations and `namespace` alias declarations, or to omit them?  That is, is the brevity worth the incompleteness?